test/integration/vm_systemd.go

```diff
@@ -27,6 +28,9 @@ import (
 
 func testVMSystemd(t *testing.T) {
 	t.Parallel()
+	if runtime.GOOS == "windows" {
```

Comment:
It is in above comment, it open a subshell that doesn't close but in other systems the output goes to the appropriate terminal and exits.